docs/mux-logging-service.md

```diff
@@ -216,7 +216,9 @@ from whichever of these it is configured to read from.
 using the ViaQ data model (i.e. the `kubernetes` and `docker` namespaces).
 
 `Filter syslog` takes the `/var/log/messages` or journald system log input and
-formats it using the ViaQ data model (i.e. the `systemd` namespace).
+formats it using the ViaQ data model (i.e. the `systemd` namespace).  **NOTE**
+reading from `/var/log/messages` is no longer supported.  `journald` is always
```

Comment:
Having the line 218's "takes the '/var/log/messages" and this "reading from '/var/log/messages' is no longer supported" confuses readers?  Is the line 218 needed for some historical reason?

(Sorry to put this comment after merged.  The above question is very minor.  If there is another chance, please consider it.)

Reply:
Yes, needed for historical purposes.